poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFRun.java

```diff
@@ -849,7 +929,22 @@ public int getFontSize() {
     }
 
     /**
-     * Specifies the font size which shall be applied to all non complex script
+     * Specifies the font size which shall be applied to the complex script
+     * characters in the contents of this run when displayed.
+     *
+     * @return value representing the font size (non-integer size will be rounded with half rounding up,
+     * -1 is returned if size not set)
+     * @deprecated use {@link #getComplexScriptFontSizeAsDouble()}
+     */
+    @Deprecated
+    @Removal(version = "6.0.0")
```

Comment:
for the new code only, could we could just `public double getComplexScriptFontSize()`? No `getComplexScriptFontSizeAsDouble` needed - let's not add new methods that have deprecated or removal annotations on them.

Reply:
Done